<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OscilloscopeBox_reader</class>
 <widget class="QWidget" name="OscilloscopeBox_reader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="layout_scope"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_filename"/>
     </item>
     <item>
      <widget class="QToolButton" name="btn_choose_flle">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_prev">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_next">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
